clients/src/main/java/org/apache/kafka/clients/consumer/internals/FetchMetricsManager.java

```diff
@@ -24,15 +24,18 @@
 import org.apache.kafka.common.metrics.stats.WindowedCount;
 
 import java.util.Collections;
-import java.util.HashMap;
 import java.util.Map;
 import java.util.Set;
 
+import static org.apache.kafka.common.utils.Utils.mkEntry;
+import static org.apache.kafka.common.utils.Utils.mkMap;
+
 /**
  * The {@link FetchMetricsManager} class provides wrapper methods to record lag, lead, latency, and fetch metrics.
  * It keeps an internal ID of the assigned set of partitions which is updated to ensure the set of metrics it
  * records matches up with the topic-partitions in use.
  */
+@SuppressWarnings("deprecation")
```

Comment:
I don't believe this suppression has any value because it's implementing deprecated code, as opposed to using it.

Reply:
Yes you are right, I have corrected it.